core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/diagnostics/DiagnosticsFlowService.java

```diff
@@ -297,4 +328,10 @@ private boolean allNetworkNodesInUnknownStatus(List<NodeStatusProto.NetworkDetai
                 .collect(Collectors.toList());
     }
 
+    public boolean isVersionGreaterOrEqual(String actualVersion, String versionToCompare) {
+        ModuleDescriptor.Version actVersion = ModuleDescriptor.Version.parse(actualVersion);
+        ModuleDescriptor.Version versionToCmp = ModuleDescriptor.Version.parse(versionToCompare);
+        return actVersion.compareTo(versionToCmp) >= 0;
+    }
```

Comment:
This certainly works, just wondering if using the CB `VersionComparator` would be better for consistency.

Reply:
i would rather use this, as it works for different version formats and the calculation won't change if the CB versioning changes for some reason